### DISCUSSION TITLE
I use v-for to render q-input with v-model, I can't update data on input UI

### DISCUSSION THREAD

**Discussion by tssghost89** [quasarframework/quasar#16597] [General - Components / Directives / etc]
### What happened?

I use v-for to render q-input with v-model, but data is not update on UI.
I can change data of q-input but after lost focus, it becomes the data before.
This is my script
![image](https://github.com/quasarframework/quasar/assets/53928563/f7c06017-2e4e-4f01-a6ad-ffc461ee448f)

![image](https://github.com/quasarframework/quasar/assets/53928563/13030cf9-da37-4f64-8d58-5d928494cb31)

UI Show:
![image](https://github.com/quasarframework/quasar/assets/53928563/b4cfa238-26dc-478b-94a7-91032817c2ae)

I change data
![image](https://github.com/quasarframework/quasar/assets/53928563/4f7cb203-cdc2-494d-92a3-69c897c8202b)

After lost focus
![image](https://github.com/quasarframework/quasar/assets/53928563/0c0e4a4d-18be-4af2-9529-603063d5f9d8)

### What did you expect to happen?

Data update on input UI

### Reproduction URL

https://github.com/tssghost89/quasar-problem-qinput

### How to reproduce?

1. Clone my github link
2. run npm i
3. run quasar dev

### Flavour

Quasar CLI with Vite (@quasar/cli | @quasar/app-vite)

### Areas

Components (quasar)

### Platforms/Browsers

Chrome

**Comment by pdanpdan** [comment]
This has nothing to do with Quasar, but with Vue reactivity.
The cheapest solution is below, but you should read the Vue docs on reactivity.

https://codepen.io/pdanpdan/pen/YzBrvVz?editors=1010

**Reply by tssghost89** [reply]
Yours works but mine not.
Please helping me check my project on github https://github.com/tssghost89/quasar-problem-qinput

![image](https://github.com/quasarframework/quasar/assets/53928563/c41b43f2-48a3-4aef-9f7b-1ed15ffd69b9)

**Reply by tssghost89** [reply]
omg, I change setup to data, it works !!!
![image](https://github.com/quasarframework/quasar/assets/53928563/bfde52ca-fde1-417c-a817-b745a7a1a2f5)
